src/runtime/components/forms/RadioGroup.vue

```diff
@@ -18,7 +18,7 @@
         @change="onUpdate(option.value)"
       >
         <template #label>
-          <slot name="label" v-bind="{ option }" />
+          <slot name="label" v-bind="{ option, selected: option.selected }" />
```

Comment:
Why do you need to inject the `selected` in every option? You could just have it globally.